content/articles/2019-08-gargle-googledrive-1-0-0.Rmarkdown

```diff
@@ -0,0 +1,91 @@
+---
+title: gargle and googledrive v1.0.0
+slug: gargle-googledrive-1-0-0
+author: Jenny Bryan
+description: >
+  gargle and googledrive v1.0.0 are now on CRAN.
+date: '2019-08-21'
+categories: [package]
+tags:
+  - gargle
+  - googledrive
+photo:
+  url: https://flic.kr/p/oe2LJ1
+  author: North Carolina Christian advocate (1894)
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>",
+  fig.width = 7,
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+# options(crayon.enabled = TRUE)
+```
+
+## Introduction
+
+We're elated to announce the initial release of gargle and of googledrive v1.0.0 on CRAN!
+
+  * The [gargle package](https://gargle.r-lib.org) is meant to take some of the pain out of working with Google APIs. It's mostly aimed at the maintainers of R packages that call Google APIs. If we're successful, users won't really notice gargle, they'll just feel like these packages got easier to work with and less idiosyncratic.
+  * The [googledrive package](https://googledrive.tidyverse.org) is a great example of such a package: it helps useRs work with Google Drive from R. The v1.0.0 release gets all auth functionality (and more) from gargle and also has a new feature that makes file upload/update more user-friendly.
```

Comment:
alphabetical order aside, i'd put googledrive first: most people will be more likely to use googledrive, and eyes glaze over fast once the word "auth" appears. :)

Reply:
I've decided to split into 2 posts, with each linking to the other. Then we can promote them separately, so the eyes-glazing-over folks can just read about googledrive.